docs/spec.md

```diff
@@ -160,10 +163,50 @@ following constraints:
 * (C12) If `quantization_dimension` is not empty, then
   `0 <= quantization_dimension`.
 
-At the moment, `QuantizationScale` is a floating-point constant, but there is
-strong interest in integer-based scales, represented with multipliers and
-shifts. We are planning to explore this in the near future
-([#1404](https://github.com/openxla/stablehlo/issues/1404)).
+In order to allow operation using only integer arithmetic, the floating-point
+`scale`, `S`, is realized using integer `multipler`, `M` and `shift` value,
+`n >= 0`, such that `round_nearest_even(S * 2^n) = M`.
```

Comment:
The n>=0 puts a lower bound on n but no constraint on the upper bound. The example code below produces n clipped to the range 1<=n<=62 and 0 <= M < (1<<31). I think it would be good if the specification could provide some constraints on the range of n and M such as these.

Reply:
@dominicsymes That makes sense. 
I have added the bounds in this line only. PTAL.